cassandra/pom.xml

```diff
@@ -27,13 +27,14 @@
     </parent>
 
     <groupId>org.apache.zeppelin</groupId>
-    <artifactId>zeppelin-cassandra_2.10</artifactId>
+    <artifactId>zeppelin-cassandra_${scala.binary.version}</artifactId>
```

Comment:
right - right now, it's being enforced to "part as 2.10, part as 2.11". I see the "binary version" as essentially the "global version", as it's used for naming - doesn't disallow you to build from source, internally, and use "Spark 1.6 w/ Scala 2.10.5"

thoughts? Again, I can make the binary.version per-package as well if you prefer, just don't feel it's really necessary